testcases/PowerNVDump.py

```diff
@@ -791,16 +791,47 @@ def runTest(self):
                 boot_type = self.kernel_crash(crash_type="hmc")
                 self.verify_dump_file(boot_type)
 
+class KernelCrash_KdumpWorkLoad(PowerNVDump):
+
+    # This test verifies kdump/fadump after running ebizzy.
+    # ebizzy url needs to be given in ~/.op-test-framework.conf.
+    # Ex: url=http://liquidtelecom.dl.sourceforge.net/project/ebizzy/ebizzy/0.3/ebizzy-0.3.tar.gz
+
+
+    def runTest(self):
+        if not self.url:
+            raise self.skipTest("Provide ebizzy url in op-test-framework.conf")
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
+        self.setup_test()
+        if self.distro == "rhel":
+            self.c.run_command("yum -y install make gcc wget; cd /tmp", timeout=120)
+        elif self.distro == "ubuntu":
+            self.c.run_command("apt-get install -y make gcc wget; cd /tmp", timeout=120)
+        else:
+            self.c.run_command("zypper install -y make gcc wget; cd /tmp", timeout=120)
+        self.c.run_command("wget %s" % self.url)
+        self.c.run_command("tar -xf ebizzy*.tar.gz")
```

Comment:
@Pavithra1602 as looking patch looks fine the only issue is if any one of commands fails next command must not works , so we need to have proper error handling , at least like wget and make

Reply:
As praveen said, better handle in a try statement